entries/t/threatlocker.com.json

```
@@ -0,0 +1,13 @@
+{
+  "ThreatLocker": {
+    "domain": "threatlocker.com",
+    "url": "https://www.threatlocker.com/",
+    "tfa": [
+      "totp"
+    ],
```

```suggestion
      "totp",
      "custom-software"
    ],
    "custom-software": [
      "Duo"
    ],
```
Looks like they also offer Duo Push [as documented here](https://threatlocker.kb.help/configuring-with-duo-authentication/).